ansible/server-state-playbooks/www/requirements.yml

```diff
@@ -16,9 +13,6 @@
   src: https://github.com/openmicroscopy/ansible-role-jekyll-build.git
   version: 1.2.0
 
-- name: openmicroscopy.lvm-partition
```

Comment:
It's used by the docker role if you choose the custom storage option https://github.com/openmicroscopy/ansible-role-docker#custom-storage